kong/plugins/cors/handler.lua

```diff
@@ -212,7 +214,6 @@ function CorsHandler:access(conf)
   end
 
   local methods = conf.methods and concat(conf.methods, ",")
-                  or "GET,HEAD,PUT,PATCH,POST,DELETE"
```

Comment:
Please do not remove the fallback here (it doesn't seem like `config.methods` is properly labelled as being required). And if it is, then we'd need to update the above assignment conditions anyway. But better stay safe here, we are shipping this patch as a fix without an RC cycle, let's not forget.